tools/c7n_azure/c7n_azure/container_host/host.py

```diff
@@ -0,0 +1,379 @@
+# Copyright 2019 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import base64
+import json
+import logging
+import os
+import tempfile
+from datetime import datetime
+
+import yaml
+from apscheduler.schedulers.blocking import BlockingScheduler
+from apscheduler.triggers.cron import CronTrigger
+from azure.mgmt.eventgrid.models import \
+    StorageQueueEventSubscriptionDestination, StringInAdvancedFilter, EventSubscriptionFilter
+from c7n_azure import entry, constants
+from c7n_azure.azure_events import AzureEventSubscription, AzureEvents
+from c7n_azure.provider import Azure
+from c7n_azure.session import Session
+from c7n_azure.storage_utils import StorageUtilities as Storage
+from c7n_azure.utils import ResourceIdParser
+
+from c7n.config import Config
+from c7n.policy import PolicyCollection
+from c7n.resources import load_resources
+from c7n.utils import local_session
+
+log = logging.getLogger("c7n_azure.container-host")
+max_dequeue_count = 2
+policy_update_seconds = 60
+queue_poll_seconds = 15
+jitter_seconds = 10
+queue_timeout_seconds = 5 * 60
+queue_message_count = 5
+
+
+class Host:
+
+    def __init__(self):
+        logging.basicConfig(level=logging.INFO, format='%(message)s')
```

Comment:
Is there a support for 'verbose' logging?

Reply:
This probably shouldn't be here anymore actually.

Reply:
So at the moment I have no additional debug logging, and there is minimal value to defaulting to anything less verbose since these logs stay in the container and are separate from the policy output, so I may leave this as is until it is useful to support a `debug` env.